### PR TITLE
Remove code for students from transaction part

### PR DESCRIPTION
**Pull request by nrc**
PTAL @Connor1996

### CODE REVIEW

File: kv/transaction/mvcc/transaction.go [hunk 1]
```diff
@@ -1,64 +1,70 @@
...
+}
+
+func (txn *MvccTxn) StartTS() uint64 {
+	return 0
+}
 
 // Writes returns all changes added to this transaction.
-func (txn *MvccTxnStub) Writes() []storage.Modify {
+func (txn *MvccTxn) Writes() []storage.Modify {
 	return nil
```
Comment (Connor1996) [inline kv/transaction/mvcc/transaction.go]: please add `// Your Code Here (4A).`

File: kv/transaction/mvcc/transaction.go [hunk 2]
```diff
@@ -1,64 +1,70 @@
...
 
 	"github.com/pingcap-incubator/tinykv/kv/storage"
 	"github.com/pingcap-incubator/tinykv/kv/util/codec"
-	"github.com/pingcap-incubator/tinykv/kv/util/engine_util"
 )
 
 // MvccTxn groups together writes as part of a single transaction. It also provides an abstraction over low-level
 // storage, lowering the concepts of timestamps, writes, and locks into plain keys and values.
-type MvccTxnStub struct{}
+type MvccTxn struct{}
```
Comment (Connor1996) [inline kv/transaction/mvcc/transaction.go]: please add `// Your Data Here (4A).`

**Comment by nrc** [comment]
I have added the comments, I accidentally pushed to upstream/course, so this PR doesn't need to be merged; sorry. If you have any more comments, I can make a follow-up PR.

**Comment by Connor1996** [comment]
LGTM overall 
https://github.com/pingcap-incubator/tinykv/commit/61e9e923c0573461c83eac47ed0e0354f9449d82#r37934103 PTAL

**Pull request closed without merging**